includes/frontend/class-wcmp-frontend.php

```diff
@@ -52,22 +49,47 @@ public function __construct()
      *
      * @throws \Exception
      */
-    public function email_pickup_html(WC_Order $order): void
+    public function confirmationEmail(WC_Order $order): void
     {
-        WCMYPA()->admin->showDeliveryDateForOrder($order);
+	    WCMYPA()->admin->showShipmentConfirmation($order, true);
     }
 
     /**
      * @param int $order_id
      *
      * @throws Exception
      */
-    public function thankyou_pickup_html(int $order_id): void
+    public function confirmationOrderReceived(int $order_id): void
     {
         $order = wc_get_order($order_id);
-        WCMYPA()->admin->showDeliveryDateForOrder($order);
+        WCMYPA()->admin->showShipmentConfirmation($order, false);
     }
 
+	/**
+	 * @param array    $actions
+	 * @param WC_Order $order
+	 *
+	 * @return array[]
+	 * @throws \Exception
+	 */
+	public function confirmationInMyAccount(array $actions, WC_Order $order): array
+	{
+		$order_id = WCX_Order::get_id($order);
+		if ($consignments = WCMP_Frontend::getTrackTraceLinks($order_id)) {
```

Comment:
assignment in een if is best wel vies, kun je er een variabele van maken en hem eruit trekken?